source/common/network/io_socket_handle_impl.cc

```diff
@@ -477,7 +533,7 @@ IoHandlePtr IoSocketHandleImpl::accept(struct sockaddr* addr, socklen_t* addrlen
     return nullptr;
   }
   return SocketInterfaceImpl::makePlatformSpecificSocket(result.return_value_, socket_v6only_,
-                                                         domain_);
+                                                         domain_, {});
```

Comment:
Why does this do something different from the similar call below? Perhaps we should have a comment?

Reply:
accept() is only used by TCP socket which doesn't use the address cache. Whereas dup() can be used by both TCP and UDP, so copy over the same creation option.